firebase_admin/db.py

```diff
@@ -199,6 +232,30 @@ def delete(self):
         """
         self._client.request_oneway('delete', self._add_suffix())
 
+    def transaction(self, transaction_update):
```

Comment:
Lets return a boolean from this method to indicate success/failure. That way the caller has a deterministic way of knowing whether the transaction was committed or not:

```
if ref.transaction(update_func):
  do_more_stuff()
else:
  raise DatabaseError('Failed to commit txn')
```